src/brief_handler.ml

```diff
@@ -70,6 +70,7 @@ let t ~args = object (self)
               ("%2F","/");("%3F","?" ); ("%3D","="); ("%26","&");
               ("%25","%");("+"," ");    ("%3E",">"); ("%3C","<");
               ("%3A",":");("&amp;","&");("&quot;","\"");
+              ("%2d","-");
```

Comment:
Since most of the other matches assume upper-case hex digits, do we need to worry about "%2D"?

Reply:
I'll add both.